setup.py

```diff
@@ -29,7 +34,8 @@ def read(filename):
 def unsphinx(text):
     # remove Sphinx extensions used in CHANGES.rst from reStructuredText
     # so that it can be handled by plain docutils
-    return text.replace(':func:', '').replace('.. currentmodule:: objgraph', '')
+    return text.replace(':func:', '').replace('.. currentmodule:: objgraph',
+                                              '')
```

Comment:
Gah you're right that is super ugly. I'll just add some parens and make it better.